gpuRIR/extensions/hrtf/hrtf_rir.py

```diff
@@ -4,7 +4,11 @@
 from matplotlib import pyplot as plt
 
 class HRTF_RIR:
+    ''' Head related transfer function room impulse response (HRFT RIR) interface.
```

Comment:
Reference: Algazi V.R., Duda R.O., Thompson D.M. and Avendano C. The CIPIC
HRTF database. In Proceedings of the 2001 IEEE Workshop on the Applications of
Signal Processing to Audio and Acoustics (Cat. No.01TH8575) (2001), pp. 99–102.